certbot-apache/certbot_apache/augeasparser.py

```diff
@@ -100,6 +99,65 @@ def __init__(self, **kwargs):
     def save(self, msg):
         self.parser.save(msg)
 
+    def find_ancestors(self, name):
+        """
+        Searches ancestor BlockNodes with a given name.
+
+        :param str name: Name of the BlockNode parent to search for
+
+        :returns: List of matching ancestor nodes.
+        :rtype: list of AugeasBlockNode
+        """
+
+        ancestors = []
+
+        parent = self.metadata["augeaspath"]
+        while True:
+            # Get the path of ancestor node
+            parent = parent.rpartition("/")[0]
```

Comment:
does augeas always specifically use "/" in an api-defined way, or should we actually be looking for something like `pathsep`?

Reply:
Yeah, Augeas always uses `/` in values returned from the searches as they always point to an absolute location.